### DISCUSSION TITLE
osu!taiko Extended Results Screen No Longer Centered

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16794] [Q&A] [answered]
Regression as of Version 2022.205.0. Possibly related to #16743? I can't really break down the details myself.
Previous Version:
![osu_2022-01-30_12-10-49](https://user-images.githubusercontent.com/74262407/152668193-5d1719c9-78f3-4d85-b761-f36c68b8577d.png)
Current Version:
![osu_2022-02-05_20-47-23](https://user-images.githubusercontent.com/74262407/152668197-ff12141e-bc17-48eb-afb0-d6defaa8b372.png)
![osu_2022-02-05_20-47-29](https://user-images.githubusercontent.com/74262407/152668198-8e893f2c-e4de-441a-b75b-ee47cb1d7e4a.png)

**Comment by peppy** [comment, marked as answer]
This is an intended change, as we add more statistics scrolling is required.

**Reply by zachmanthethird** [reply]
Just curious if fewer statistics would recenter vertically as opposed to staying top-justified :+1:
